downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -192,6 +238,10 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
 
 ### New features
 
+#### New Runtime Java APIs
+
+- Introduced the `Type getImpliedType(Type)` API in the `io.ballerina.runtime.api.utils.TypeUtils` class to recursively retrieve the referred type or the effective type of given type.
```

Comment:
```suggestion
- Introduced the `Type getImpliedType(Type)` API in the `io.ballerina.runtime.api.utils.TypeUtils` class to retrieve the the implied type for a given type, obtained by recursively resolving type reference and/or intersection types.
```

Reply:
Can't we add an example here?

Reply:
Added an example [ee848e2](https://github.com/ballerina-platform/ballerina-dev-website/pull/7606/commits/ee848e2560da5b858aa68eae18bfbdbe3c252d67). I prefer the existing description replaced `or` with `and/or`

Reply:
"referred" and "effective" are internal terms that may not be obvious to the user, unlike type reference and intersection types.

Reply:
I don't think we can expect a user to read the entire release note.

Reply:
The thing is what do we mean by resolving intersection? It does not give us the meaning of `corresponding non-intersection type that is computed to represent the same value space`. One thing we can do is add `abbreviation` section to the release note or like reference section since we have used the term `effectiveType` in 3 places, but only the first occurrence contains the definition.

Reply:
How about

```md
- Introduced the `Type getImpliedType(Type)` API in the `io.ballerina.runtime.api.utils.TypeUtils` class to retrieve the implied type for a given type, obtained by recursively resolving type reference types (to get referred types) and/or intersection types (to get effective types).
```